test/LocalizationWebsite/LocalizationWebsite.csproj

```diff
@@ -21,4 +21,10 @@
     <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="1.2.0-*" />
   </ItemGroup>
 
+  <ItemGroup>
+    <EmbeddedResource Update="Resources\StartupCustomCulturePreserved.en-US.resx">
+      <Generator>ResXFileCodeGenerator</Generator>
+    </EmbeddedResource>
+  </ItemGroup>
```

Comment:
Is this because no generator is wired up by default (wow, if so)? Or, does this resx need a specific generator?